_rules/printable-characters-shortcut-ffbc54.md

```diff
@@ -0,0 +1,483 @@
+        padding: 1em;
+        padding-top: 0em;
+      "
+		  id="overlay"
+	  >
+      <p>Disable/remap shortcut</p>
+      <label>
+        <input type="checkbox" onclick="toggleDisabled('singleShortcut', !this.checked)" checked>
+        Toggle single character keyboard shortcut
+      </label>
+      <br>
+      <label>
+        <input id="remap" type="checkbox" onclick="toggleModifier('singleShortcut', this.checked)">
+        Use "ctrl" key together with the "+" key
+      </label>
+      <br>
+      <button onclick="closeModal();">Dismiss</button>
+    </div>
+    <p>To control the shortcuts activate the "Control shortcuts" button.</p>
+    <input type="button" onclick="openModal()" value="Control shortcuts" />
+    
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+### Failed
+
+#### Failed Example 1
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, but there is no [instrument][] to **remap** the [keyboard event][] and the [keyboard event][] is not [blocked][blocked event] when the [event target][] does not have [focus][].
+
+```html
+<html>
+  <head>
+    <title>Failed Example 1</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+  </head>
+  <body onload="registerShortcut({shortcutKey: '+', disabled: false}); activateShortcuts();">
+    <label for="target">Add to list (press "+" to add):</label>
+    <input type="text" id="target" />
+    <br />
+    <div>
+      To do list
+    </div>
+    <ul id="list"></ul>
+  </body>
+</html>
+```
+
+#### Failed Example 2
+
+This [HTML document][] has a [keyboard event][] [dispatched][] to an [event target][] with the attribute `key` being a [printable character][] and the method `getModifierState` returning `false`, and there is an [instrument][] to disable the [keyboard event][] and another [instrument][] to **remap** the [keyboard event][] so that it requires that the method `getModifierState` returns `true`, but the [instruments][instrument] are not in a [clearly labeled location][].
+
+```html
+<html>
+  <head>
+    <title>Failed Example 2</title>
+    <script src="/test-assets/ffbc54/shortcut.js"></script>
+    <script>
+      function openModal() {
+        document.getElementById('overlay').style.display = 'block'
+      }
+      function closeModal() {
+        document.getElementById('overlay').style.display = 'none'
+      }
+    </script>
+  </head>
+  <body onload="registerShortcut({id: 'singleShortcut', shortcutKey: '+'}); activateShortcuts();">
+    <div
+      style="
+        display: none;
+        position: fixed;
+        top: 2em;
+        left: 10em;
+        background-color: #505050;
+        color: white;
+        padding: 1em;
+        padding-top: 0em;
+      "
```

Comment:
Same remark about JS and style. The fact that it is shared makes it even more valuable to share it.